src/encoded/audit/experiment.py

```diff
@@ -1315,11 +1327,21 @@ def check_experiment_small_rna_standards(experiment,
                               'MadQualityMetric',
                               desired_assembly,
                               desired_annotation)
+    replicates = experiment.get('replicates')
 
-    yield from check_spearman(
-        mad_metrics, experiment['replication_type'],
-        0.9, 0.8, 'Small RNA-seq single-end pipeline')
-    return
+    if experiment['replication_type'] == 'unreplicated' and len(replicates) > 1:
+        yield from check_spearman_technical_replicates(
+            mad_metrics, 'Small RNA-seq single-end pipeline',
+            0.9)
+
+        return
+
+    else:
+        yield from check_spearman(
+            mad_metrics, experiment['replication_type'],
+            0.9, 0.8, 'Small RNA-seq single-end pipeline')
+
+        return
```

Comment:
I think the technical MAD should be limited to bulk RNA only.
we are not planning to run small RNA-seq single-end pipeline (it is the old RNA pipeline, I don't think it has to handle the tech rep MAD case)